pkg/pipeline/extract/aggregate/aggregate_test.go

```diff
@@ -116,8 +118,8 @@ func Test_Evaluate(t *testing.T) {
 	err := aggregate.Evaluate(entries)
 
 	require.Equal(t, err, nil)
-	require.Equal(t, aggregate.Groups[normalizedValues].totalCount, 2)
-	require.Equal(t, aggregate.Groups[normalizedValues].totalValue, float64(7))
+	require.Equal(t, aggregate.GroupsMap[normalizedValues].totalCount, 2)
+	require.Equal(t, aggregate.GroupsMap[normalizedValues].totalValue, float64(7))
```

Comment:
You can also add here a test on the length of `GroupsList`.

Reply:
done